libraries/AP_Mission/AP_Mission.h

```diff
@@ -244,7 +244,7 @@ class AP_Mission
         // do-guided-limits
         Guided_Limits_Command guided_limits;
 
-        // cam trigg distance
+        // high altitude balloon altitude wait
```

Comment:
why balloon?

Reply:
it was originally added for balloon drops